### ISSUE TITLE
A11y: [New room] Name and Topic fields lack programmatic labels....

### ISSUE THREAD

**Issue by mxandreas** [element-hq/element-x-ios#5257]
- **Checkpoint ID:** 11.1.3.1d/6.1
- **View/Component:** New room

### Description
Name and Topic fields lack programmatic labels.

### Recommendation
Associate visible labels with fields; don't rely on placeholder.

--- 
*Report Date: 20.02.2026*
